specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/artifacts.json

```diff
@@ -4177,6 +4499,16 @@
         "type",
         "referenceName"
       ]
+    },
+    "ArtifactRenameRequest": {
+      "description": "Request body structure for rename artifact.",
+      "type": "object",
+      "properties": {
+        "newName": {
+          "description": "New name of the artifact.",
+          "type": "string"
```

Comment:
The original name in the `PUT` has constraints:

``` json 
      "pattern": "^[A-Za-z0-9_][^<>*#.%&:\\\\+?/]*$",
      "minLength": 1,	      "minLength": 1,
      "maxLength": 260,	      "maxLength": 260,
```

I'm assuming that the new name should have the same constraints, right?

(another good reason to have a common parameter definition, so that these are consistent)